build/Microsoft.DotNet.Cli.Prepare.targets

```diff
@@ -128,6 +132,7 @@
     <PropertyGroup>
       <CoreSetupBlobRootUrl>https://dotnetcli.azureedge.net/dotnet/$(CoreSetupChannel)</CoreSetupBlobRootUrl>
       <SharedFrameworkArchiveBlobRootUrl>$(CoreSetupBlobRootUrl)/Binaries/$(SharedFrameworkVersion)</SharedFrameworkArchiveBlobRootUrl>
+      <NuGetPackagesArchiveBlobRootUrl>$(CoreSetupBlobRootUrl)/Binaries/$(NuGetPackagesArchiveVersion)</NuGetPackagesArchiveBlobRootUrl>
```

Comment:
Does this mean that we are going to have potentially a bunch of weird folders in the blob? I wonder if we should just keep adding the lzma archives in the shared framework version to which they belong. Just change the lzma filename itself with the hash.

What do you think?